reactive/kubernetes_master.py

```diff
@@ -3639,7 +3653,13 @@ def configure_kubelet():
 
     registry = hookenv.config("image-registry")
     taints = hookenv.config("register-with-taints").split()
-    kubernetes_common.configure_kubelet(dns_domain, dns_ip, registry, taints=taints)
+    try:
+        kubernetes_common.configure_kubelet(
+            dns_domain, dns_ip, registry, taints=taints, has_xcp=has_xcp
+        )
+    except TypeError:
+        # older kubernetes-common doesn't support the has_xcp flag
+        kubernetes_common.configure_kubelet(dns_domain, dns_ip, registry, taints=taints)
```

Comment:
This was just trying to avoid integration test failures while the dependent PRs are as yet to be merged. But since they're failing anyway, I might as well remove this ugliness.